plugin/src/main/java/com/denizenscript/denizen/utilities/BukkitImplDeprecations.java

```diff
@@ -268,6 +268,9 @@ public class BukkitImplDeprecations {
     // Added 2025/03/14
     public static Warning settingBoatType = new Warning("settingBoatType", "As of MC 1.21, separate boat wood types are separate entity types, meaning the wood type of an existing boat entity cannot be changed without spawning a new one.");
 
+    // Added 2025/07/10
+    public static Warning entityKnockback = new Warning("entityKnockback", "The 'entity.knockback' property is deprecated. You should adjust knockback enchantment on the weapon itself.");

```

Comment:
```suggestion
    public static Warning entityKnockback = new Warning("entityKnockback", "The 'EntityTag.knockback' property is deprecated. You should adjust the knockback enchantment on the weapon itself.");
```
Otherwise all LGTM.